seeding/index.js

```diff
@@ -0,0 +1,64 @@
+import chalk from 'chalk';
+import { startDBConnection } from '../server/config';
+import logger from '../server/config/winston';
+// eslint-disable-next-line import/no-cycle
+import seedUsers from './user.seeding';
+// eslint-disable-next-line import/no-cycle
+import resetDB from './reset.seeding';
+
+export const logSuccess = (data) => {
+  logger.info(chalk.greenBright(data));
+};
+
+export const logLoading = (data) => {
+  logger.info(chalk.yellowBright(data));
+};
+
+export const logInfo = (data) => {
+  logger.info(chalk.blueBright(data));
+};
+
+export const logError = (data) => {
+  logger.error(chalk.redBright(data));
+};
+
+const MODEL = {
+  USER: 'user',
```

Comment:
Lets ensure that that the values of the `env` variables we are getting from the user is in uppercase for consistency sake:

```suggestion
  USER: 'USER',
```

Please update the one below too for `RESET`